src/zcl/definition/cluster.ts

```diff
@@ -5166,7 +5166,262 @@ const Cluster: {
             },
         },
         commandsResponse: {},
-    },    
+    },
+    manuSpecificAssaDoorLock: {
+        ID: 0xFC00,
```

Comment:
Could we add a manufacturer code here like; https://github.com/Koenkk/zigbee-herdsman/blob/064662f37da1c0538e52e09f80bc3f724eb693a9/src/zcl/definition/cluster.ts#L4141

Reply:
I looked for one in the decrypted driver and sniffs but couldn't find one. Will dig a bit deeper

Reply:
Just double checked the sniff, and there isn't any `manufacturerCode` that is sent along with the packets..?

![image](https://drive.google.com/uc?export=view&id=1PKAuK-9mmktushwiQ5rslWPn8RqgtC-_)

Reply:
Thanks for checking!